src/Service/GitLabClient.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Service;
+
+use App\Badge\Exception\RepositoryDataNotValid;
+use Gitlab\Client;
+
+final class GitLabClient implements GitLabClientInterface
+{
+    public function __construct(private string $gitLabToken, private Client $gitlabClient)
```

Comment:
I prefer before the object deps and after string etc, WDYT?

Reply:
Yes, I agree.

Reply:
Fixed.